cartridges/rvw_dev_console/cartridge/client/default/js/vue/components/App.vue

```diff
@@ -740,28 +695,31 @@ export default {
                 localStorage.setItem('lastRun', JSON.stringify(this.code));
 
                 try {
-                    const response = await this.axios.post(`${window.urlPath}/Console-Run`, data);
+                    var self = this;
+                    const response = await this.axios.post(`${window.urlPath}/Console-Run`, data).catch(function(error){
```

Comment:
@z1haze I added this catch block here since we could actually capture the HTTP response this way, vs the other catch that was just a generic HTTP error that was getting triggered if the response was not 200 status.  This solution will show the JSON response the server returned.

Reply:
if you use an arrow function you dont have to do the hacky self = this thing :D